scripts/website/dump_info/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/golangci/golangci-lint/pkg/config"
+	"github.com/golangci/golangci-lint/pkg/lint/lintersdb"
+	"github.com/golangci/golangci-lint/scripts/website/shared"
+)
+
+func main() {
+	linters := lintersdb.NewLinterBuilder().Build(config.NewDefault())
+
+	var wraps []shared.LinterWrapper
+	for _, l := range linters {
+		wraps = append(wraps, shared.LinterWrapper{Config: l, Name: l.Linter.Name(), Desc: l.Linter.Desc()})
+	}
+
+	err := saveToJSONFile(filepath.Join("assets", "linters-info.json"), wraps)
+	if err != nil {
+		log.Fatalf("save linters: %v", err)
```

Comment:
I propose to be consistent and start `log` messages from the capital letter:

```suggestion
		log.Fatalf("Save linters: %v", err)
```

The comment applies to log messages in the whole file.